packages/flame/lib/src/parallax.dart

```diff
@@ -134,18 +241,18 @@ class ParallaxLayer {
     double scale(LayerFill fill) {
       switch (fill) {
         case LayerFill.height:
-          return parallaxImage.image.height / size.y;
+          return parallaxRenderer.image.height / size.y;
         case LayerFill.width:
-          return parallaxImage.image.width / size.x;
+          return parallaxRenderer.image.width / size.x;
         default:
           return _scale;
       }
     }
 
-    _scale = scale(parallaxImage.fill);
+    _scale = scale(parallaxRenderer.fill);
```

Comment:
it seems _scale calls scale that then calls _scale back? I got quite confused here